crates/ruff_linter/src/rules/flake8_datetimez/snapshots/ruff_linter__rules__flake8_datetimez__tests__DTZ007_DTZ007.py.snap

```diff
@@ -1,47 +1,50 @@
 ---
 source: crates/ruff_linter/src/rules/flake8_datetimez/mod.rs
 ---
-DTZ007.py:4:1: DTZ007 The use of `datetime.datetime.strptime()` without %z must be followed by `.replace(tzinfo=)` or `.astimezone()`
+DTZ007.py:4:1: DTZ007 Naive datetime constructed using `datetime.datetime.strptime()` without %z
   |
 3 | # bad format
 4 | datetime.datetime.strptime("something", "%H:%M:%S%Z")
   | ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ DTZ007
 5 | 
 6 | # no replace or astimezone
   |
+  = help: Call `.replace(tzinfo=)` or `.astimezone()` to convert to an aware datetime
 
-DTZ007.py:7:1: DTZ007 The use of `datetime.datetime.strptime()` without %z must be followed by `.replace(tzinfo=)` or `.astimezone()`
+DTZ007.py:7:1: DTZ007 Naive datetime constructed using `datetime.datetime.strptime()` without %z
   |
 6 | # no replace or astimezone
 7 | datetime.datetime.strptime("something", "something")
   | ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ DTZ007
 8 | 
 9 | # wrong replace
   |
+  = help: Call `.replace(tzinfo=)` or `.astimezone()` to convert to an aware datetime
```

Comment:
Nit: I don't know if there's precedence for "incomplete" syntax in other help texts. I recommend changing it to one of

```
.replace(tzinfo=...) 
.replace(tzinfo=<timezone>)
```

to make it clear, that a value must follow after the `=`